plumbum/cli/progress.py

```diff
@@ -139,16 +139,16 @@ def __str__(self):
         if width - len(ending) < 10 or self.has_output:
             self.width = 0
             if self.timer:
-                return "{0:g}% complete: {1}".format(100*percent, self.str_time_remaining())
+                return "{0:%} complete: {1}".format(percent, self.str_time_remaining())
```

Comment:
This is slightly different, this returns 0.000000% instead of 0% (etc), it should be: `{0:.0%}` to be more reasonable.